### ISSUE TITLE
Import default support

### ISSUE THREAD

**Issue by shaka3** [kei-ito/rollup-plugin-glob-import#9]
> main.js
```javascript
import textEncode from 'textEncode';

const bytes = textEncode('encode this', 'cp1250');
console.log(bytes);
```
---

> textEncode.js
```javascript
import encoders from 'encoders/*';

function textEncode(str, charset) {
   return encoders[charset](str);
}

export default textEncode;
```
----

> encoders/cp1250.js

```javascript
// export other stuffs here 

function cp1250(str) {
 // blabla
}
export default cp1250;

```
---

> encoders/cp1251.js

```javascript
// export other stuffs here

function cp1251(str) {
 // blabla
}
export default cp1251;

```

**Comment by kei-ito** [comment]
OK, I'm now preparing the next release that supports `"export default X"`.
It is "Export default support", not "Import default support".

I added your sample to test: [e7ad766/test/projects/issue9](https://github.com/kei-ito/rollup-plugin-glob-import/tree/e7ad7669b0c50497fe61916d372deebab570246a/test/projects/issue9).
Note: *I changed [the first line of textEncode.js](https://github.com/kei-ito/rollup-plugin-glob-import/blob/e7ad7669b0c50497fe61916d372deebab570246a/test/projects/issue9/src/textEncode.js#L1)*.

```javascript
import * as encoders from './encoders/*';
```

Please use `import * as X from Y` to import all modules.

**Comment by kei-ito** [comment]
I published v0.1.2 includes the updates.

**Comment by shaka3** [comment]
Thank you very much!
